spread.yaml

```diff
@@ -259,6 +259,24 @@ suites:
         restore-each: |
             $TESTSLIB/reset.sh --reuse-core
 
+    tests/regression/:
+        summary: Regression tests for snapd
+        systems: [-ubuntu-core-16-64-fixme]
+        prepare: |
+            . $TESTSLIB/prepare.sh
+            if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+                prepare_all_snap
```

Comment:
I didn't know how to block the external backend, I listed all core systems mentioned in spread.yaml. Is that sufficient?